### ISSUE TITLE
Towards tuning-free DQNs

### ISSUE THREAD

**Issue by bigaidream** [bigaidream-projects/qan#7]
In addition to hyperparameters of the CNN, there are some super-parameters of the DQNs, such as the learning rates of the DQN and the coefficient of the meta-momentum. 
It seems that the overall performance is not that sensitive to these super-parameters, as long as they are reasonable enough. I'm not that keen on using Bayesian optimization to tune these super-parameters like [Scalable Bayesian Optimization Using Deep Neural Networks](http://arxiv.org/abs/1502.05700)

Thus far, these super-parameters are hard-coded by set it in proportion to the gradients of CNN. We should make them adaptive. Compared to CNN, the number of parameters of the DQN is small. 

1. We can update the DQN in a Bayesian approach.
2. We can use very complicated second-order optimization methods for DQNs (https://arxiv.org/abs/1606.04838)

**Comment by bigaidream** [comment]
Compared to CNN, the number of parameters of the DQN is small. So maybe we can update the DQN in a Bayesian approach.

**Comment by bigaidream** [comment]
Not practical at this moment.
